internal/Picker/Picker.module.less

```diff
@@ -26,6 +25,14 @@
 		margin: @agate-picker-margin;
 		padding: @agate-picker-padding;
 
+		.sizingPlaceholder {
```

Comment:
![image](https://user-images.githubusercontent.com/4239873/94495990-27896900-022e-11eb-8380-fc5b775aefc4.png)

`backgounrd-image` is applied to `.sizingPlaceholder `. Could you remove it?

Reply:
🆗